Content.Server/GameObjects/Components/Arcade/RandomArcadeGameComponent.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Robust.Server.Interfaces.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.Interfaces.Random;
+using Robust.Shared.IoC;
+using Robust.Shared.Random;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.GameObjects.Components.Arcade
+{
+    [RegisterComponent]
+    public class RandomArcadeGameComponent : Component, IMapInit
+    {
+        public override string Name => "RandomArcade";
+
+        public void MapInit()
+        {
+            var arcades = new[]
+            {
+                "BlockGameArcade",
+                "SpaceVillainArcade"
+            };
+
+            var entityManager = IoCManager.Resolve<IEntityManager>();
+
+            entityManager.SpawnEntity(
```

Comment:
Mhh idk if there is a way to just add components instead of spawning a new entity

Reply:
tried that and no, that sucks. this is the more robust & in my opinion better way of doing this